infra/label_evaluator_step_functions/lambdas/unified_receipt_evaluator.py

```
@@ -1003,30 +1003,88 @@ async def run_geometric() -> tuple[dict, float]:
+                elif use_chroma_cloud:
+                    logger.warning(
+                        "CHROMA_CLOUD_ENABLED=true but "
+                        "CHROMA_CLOUD_API_KEY is empty; "
+                        "falling back to S3 snapshot"
+                    )
+
+                if chroma_client is None and chromadb_bucket:
+                    try:
+                        chroma_root = os.environ.get(
                             "RECEIPT_AGENT_CHROMA_PERSIST_DIRECTORY",
                             "/tmp/chromadb",
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/chromadb"

(S108)